slack2hipchat.coffee

```diff
@@ -2,13 +2,14 @@
 #   A way to communicate between Slack and HipChat users. 
 
 HIP_CHAT_AUTH_TOKEN =  process.env.HIP_CHAT_AUTH_TOKEN
+HIP_CHAT_ROOM_EMOJI = process.env.HIP_CHAT_ROOM_EMOJI or ":snake:"
 SLACK_WEBHOOK_URL = process.env.SLACK_WEBHOOK_URL
-ROOM_NAME = 'sf'
+ROOM_NAME = process.env.SLACK_2_HIPCHAT_ROOM or "sf"
 SIGNATURE_PREFIX = "s2h"
 
 module.exports = (robot) ->
-  robot.hear ///^(?!(\(#{SIGNATURE_PREFIX}\))).*$///i, (msg) ->
-    if msg.envelope.room == 'sf'
+  robot.hear ///^(?!(\(|:#{SIGNATURE_PREFIX}\)|:)).*$///i, (msg) ->
```

Comment:
I don't think `:` is a regex special, but I'll escape it anyway. Wouldn't hurt. Definitely needed groups.